app/namespace.go

```diff
@@ -469,519 +875,134 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
-							},
-						},
-						{
-							Name:    "rename",
-							Usage:   "Update the name of an existing custom search attribute",
-							Aliases: []string{"rn"},
-							Flags: []cli.Flag{
-								NamespaceFlag,
-								RequestIDFlag,
-								ResourceVersionFlag,
-								&cli.StringFlag{
-									Name:     "existing-name",
-									Usage:    "The name of an existing search attribute",
-									Aliases:  []string{"en"},
-									Required: true,
-								},
-								&cli.StringFlag{
-									Name:     "new-name",
-									Usage:    "The new name for the search attribute",
-									Aliases:  []string{"nn"},
-									Required: true,
-								},
-							},
-							Action: func(ctx *cli.Context) error {
-								n, err := c.getNamespace(
-									ctx.String(NamespaceFlagName),
-								)
-								if err != nil {
-									return err
-								}
-								existingName := ctx.String("existing-name")
-								if _, exists := n.Spec.SearchAttributes[existingName]; !exists {
-									return fmt.Errorf("search attribute with name '%s' does not exist", ctx.String("existing-name"))
-								}
-								newName := ctx.String("new-name")
-								if _, exists := n.Spec.SearchAttributes[newName]; exists {
-									return fmt.Errorf("search attribute with new name '%s' already exists", ctx.String("new-name"))
-								}
-								y, err := ConfirmPrompt(ctx, "renaming search attribute may cause failures if any worker is still using the old name of the search-attributes. confirm rename?")
-								if err != nil || !y {
-									return err
-								}
-								return c.renameSearchAttribute(ctx, n, existingName, newName)
-							},
-						},
+					Action: func(ctx *cli.Context) error {
+						n, err := c.getNamespace(
+							ctx.String(NamespaceFlagName),
+						)
+						if err != nil {
+							return err
+						}
+						existingName := ctx.String("existing-name")
+						if _, exists := n.Spec.SearchAttributes[existingName]; !exists {
+							return fmt.Errorf("search attribute with name '%s' does not exist", ctx.String("existing-name"))
+						}
+						newName := ctx.String("new-name")
+						if _, exists := n.Spec.SearchAttributes[newName]; exists {
+							return fmt.Errorf("search attribute with new name '%s' already exists", ctx.String("new-name"))
+						}
+						y, err := ConfirmPrompt(ctx, "renaming search attribute may cause failures if any worker is still using the old name of the search-attributes. confirm rename?")
+						if err != nil || !y {
+							return err
+						}
+						return c.renameSearchAttribute(ctx, n, existingName, newName)
 					},
 				},
 			},
 		},
-	}, nil
+	}
+
+	// ----------- Commands for private review feature, only available when feature flag turns on -----------
+	featureFlagConfigDir := GetFeatureFlagConfigPath()
+	jsonData, err := GetFeatureFlagConfig(featureFlagConfigDir)
+	if err != nil {
+		return CommandOut{}, err
+	}
+
+	// Export Command
+	if jsonData[EnableExportFeatureFlag.Name] {
```

Comment:
[Nit] suggest separating tcld flag names and feature flag key values in the dictionary